src/documents/serialisers.py

```diff
@@ -527,6 +602,21 @@ def update(self, instance: Document, validated_data):
             instance.save()
         if "created_date" in validated_data:
             validated_data.pop("created_date")
+        if instance.custom_fields.count() > 0 and "custom_fields" in validated_data:
+            for custom_field_instance in instance.custom_fields.filter(
+                field__data_type=CustomField.FieldDataType.DOCUMENTLINK,
+            ):
+                if custom_field_instance.field not in [
```

Comment:
I think this is rebuilding the list each iteration of the `for` loop.  It could be built once, before looping, right?

Reply:
Great point, thanks